napari/utils/_proxies.py

```diff
@@ -74,16 +74,6 @@ def _private_attr_warning(name: str, typ: str) -> None:
             stacklevel=3,
         )
 
-        # This is code prepared for a moment where we want to block access to private attributes
-        # raise AttributeError(
-        #     trans._(
-        #         "Private attribute set/access ('{typ}.{name}') not allowed in this context.",
-        #         deferred=True,
-        #         name=name,
-        #         typ=typ,
-        #     )
-        # )
```

Comment:
This also needs to be kept. It is code to be triggered when a person tries to access a private attribute.

Reply:
This was commented out in the original code so I removed. Adding back in without a comment and will see the test behavior.

Reply:
No. This is code that was prepared for a moment when we finally cut the access to private attributes to give user a clear message.